### DISCUSSION TITLE
understanding-the-dotnet-ecosystem-the-evolution-of-dotnet-into-dotnet-7/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#126] [Announcements]
# understanding-the-dotnet-ecosystem-the-evolution-of-dotnet-into-dotnet-7/

In this article, previously part of my new book, we look at the introduction of .NET Core, why it was created, and how it has evolved into .NET 7.

https://andrewlock.net/understanding-the-dotnet-ecosystem-the-evolution-of-dotnet-into-dotnet-7/

**Comment by frankely** [comment]
great article, thank you for sharing.

**Comment by reznextsolutions** [comment]
Very informative and useful. One suggestion related to figure 2. Can you also add blazor framework under web technologies?

**Comment by RicardoGuzmanVelasco** [comment]
Well, in the meantime, unlike Figure 2 says, Unity engine still operates with .NET Framework 4.8 or Standard 2.0, and so will it for a few years more according one of its lead architects. Sights...

Thanks for the text!

**Reply by danielwarddev** [reply]
Far and away one of my biggest gripes with Unity right now. On top of that, the new API for Godot 4.0 now uses on the .NET 6 runtime instead of Mono.

**Comment by danielwarddev** [comment]
A nice layout of the differences between the different historical framework names, particularly for those without as much experience in .NET land. I do feel bad for people sometimes when I see posts online from people who are completely confused about what version to use when trying to learn something in C#, and sometimes they're even using 4.8! Plus, the newer libraries (like EF and ASP.NET) kept the Core moniker, like you mentioned, even though they are "graduated" from .NET Core.

**Comment by sasakrsmanovic** [comment]
Great article!

If I may :), one way to improve it, and to be in spirit of open source .net, and not pure Microsoft '.net'  , would be to highlight some other frameworks that have evolved around it, and in direct support of some of the cross-platform targets you mention. i.e. https://github.com/unoplatform/uno or other projects which are part of .NET Foundation. Full disclosure, I'm on Uno Platform team.

**Comment by chrisxfire** [comment]
> With this timeline, you know how long a version of .NET will be supported. If you use an STS track release (such as .NET 7) you know you will be supported until six months after the release of .NET 8 in November 2023. As an LTS release, .NET 8.0 will be supported until *November 2025*.

Until November 2026, no?  36 months?

**Comment by Kapusch** [comment]
That’s a very a nice post, thanks for all the details @andrewlock ! Also I did want to raise a possible typo here:
> .NET 8.0 will be supported until November 2025.

Since you mentioned LTS is 3 years, isn’t it supported until November 2026 ?

**Comment by myo-min-tin** [comment]
The .NET ecosystem without reporting or report viewer.

**Comment by imariom** [comment]
I have been far from .NET ecosystem for long time, this week I started to research it and this post summarized every change and feature any .NET developer needed to know. Many many thank you for this GREAT article.
